sklearn/decomposition/tests/test_online_lda.py

```diff
@@ -142,21 +142,6 @@ def test_lda_fit_transform(method):
     assert_array_almost_equal(X_fit, X_trans, 4)
 
 
-def test_lda_partial_fit_dim_mismatch():
-    # test `n_features` mismatch in `partial_fit`
-    rng = np.random.RandomState(0)
-    n_components = rng.randint(3, 6)
```

Comment:
I removed this test because it does exactly what the common test `check_estimators_partial_fit_n_features` does.